diff --git a/mingw-w64-libssh/PKGBUILD b/mingw-w64-libssh/PKGBUILD
index ab9baea02..a6bac8eda 100644
--- a/mingw-w64-libssh/PKGBUILD
+++ b/mingw-w64-libssh/PKGBUILD
@@ -60,7 +60,8 @@ build() {
     -DWITH_GSSAPI=OFF \
     -DUNIT_TESTING=ON \
     -DBUILD_STATIC_LIB=ON \
-    -DBUILD_SHARED_LIBS=ON \
+    -DCMAKE_C_FLAGS=-DLIBSSH_STATIC
+    -DBUILD_SHARED_LIBS=OFF \
     ../${_realname}-${pkgver}
 
   make all
