diff --git a/mingw-w64-libssh/PKGBUILD b/mingw-w64-libssh/PKGBUILD
index ab9baea02..ab219356e 100644
--- a/mingw-w64-libssh/PKGBUILD
+++ b/mingw-w64-libssh/PKGBUILD
@@ -58,9 +58,10 @@ build() {
     -DCMAKE_INSTALL_PREFIX=${MINGW_PREFIX} \
     ${extra_config} \
     -DWITH_GSSAPI=OFF \
-    -DUNIT_TESTING=ON \
+    -DUNIT_TESTING=OFF \
     -DBUILD_STATIC_LIB=ON \
-    -DBUILD_SHARED_LIBS=ON \
+    -DWITH_EXAMPLES=OFF \
+    -DBUILD_SHARED_LIBS=OFF \
     ../${_realname}-${pkgver}
 
   make all
